### PR TITLE
Add new tray icons for macOS

### PR DESCRIPTION
**Pull request by rak-phillip**
This adds new template icons for use in the tray for macOS. 

I received an icon set that represents different states (one for starting, started, stopping, stopped, and error), and I noticed that we don't necessarily have corresponding icons in windows. We don't really make use of stopped and stopping anywhere in the project, but I was thinking that it might be a good idea to get the remaining icons now for ease of use. 

Alternatively, we can remove stopped and stopping from the macOS collection.  

#236

### CODE REVIEW

**Comment by rak-phillip** [comment]
Here's some examples of the new icons. I tried to include a blue-ish background to make a fair comparison

Screen Shot 2021-09-02 at 3 39 49 PM

Screen Shot 2021-09-02 at 3 44 49 PM

Screen Shot 2021-09-02 at 3 45 27 PM

**Comment by jandubois** [comment]
> We don't really make use of stopped and stopping anywhere in the project

I think the problem is that "stopping" is taking a very short amount of time, and then the app exits, so there is no way to show the "stopped" state.

I was going to create an issue about our mental app model (do we have one or two apps (gui and background/helper/...), and what is the relationship between them from the user's point of view (like what does it mean to stop one vs. the other). But I haven't been able to clarify my own thinking about it enough to actually write it up.

File: src/main/tray.ts [hunk 1]
```diff
@@ -51,8 +52,30 @@ export class Tray {
 
   protected kubernetesState = State.STOPPED;
 
+  private isMacOs = () => {
+    return os.platform() === 'darwin';
+  }
+
+  private trayIconsMacOs = {
```
Comment (mook-as) [inline src/main/tray.ts:59]: I assume this should be `readonly` as well?

**Review by mook-as — DISMISSED** [review]
Please add the [DCO trailer](https://github.com/rancher-sandbox/rancher-desktop/pull/577/checks?check_run_id=3500978087).  The run logs has instructions.

File: src/main/tray.ts (hunk 1 above)
Comment (rak-phillip) [reply src/main/tray.ts]: I agree - updated to include the `readonly` modifier

**Review by mook-as — APPROVED** [review]

**Review by mattfarina — APPROVED** [review]
I like the stopping and stopped ones. I don't know if we'll have a case where we can show stopped. We can investigate that in the future.

These look better than what we have today.

**Pull request merged**